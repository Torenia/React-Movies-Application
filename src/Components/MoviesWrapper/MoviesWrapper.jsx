import styled from 'styled-components';

export const MoviesListWrapper = styled.div`
  background-color: #232323;
`;

export const MoviesResultsWrapper = styled(MoviesListWrapper)`
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  padding: 0;
  
  @media(max-width: 900px)  {
    justify-content: space-around
  };
`;
