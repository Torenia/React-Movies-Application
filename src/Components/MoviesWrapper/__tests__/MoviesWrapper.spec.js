import React from 'react';

import { MoviesListWrapper, MoviesResultsWrapper } from '../MoviesWrapper';

describe('<MoviesListWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MoviesListWrapper />);
  });

  it('should render a movies wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

describe('<MoviesResultsWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MoviesResultsWrapper />);
  });

  it('should render a movies results wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
