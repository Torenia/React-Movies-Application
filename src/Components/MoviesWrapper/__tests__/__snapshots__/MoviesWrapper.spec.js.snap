// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviesListWrapper /> should render a movies wrapper 1`] = `
.c0 {
  background-color: #232323;
}

<div
  className="c0"
/>
`;

exports[`<MoviesResultsWrapper /> should render a movies results wrapper 1`] = `
.c0 {
  background-color: #232323;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0;
}

@media (max-width:900px) {
  .c0 {
    -webkit-box-pack: space-around;
    -webkit-justify-content: space-around;
    -ms-flex-pack: space-around;
    justify-content: space-around;
  }
}

<div
  className="sc-AxjAm c0"
/>
`;
